### PR TITLE
Gravedigger Nature Healing + Mason + Miner Update

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

Adds a new set of sprites for the Shovels, Fixes their allignments and Re Adds the Chopping attack for the Gravediggers + adds them average skill with polearms (they are people who lives using that tool)

 (the chopping attack is the same as an stone axe for balance reasons, iron axe chopping made it be an iron axe as a tool and wasn't fair)

The mason Gets A Drippy Leather Vest, Lamptern, Flint,  hunting Knife to work Better at gathering materials since they are one main building role and not a towner with less tools

The Miner Gets a Lamptern To Illuminate their Ways (be on the mines is his profesion, why they never had a Lamp with them?)

## Why It's Good For The Game

Fixes an issue we had for a long time, adds more sprite quality and adds some fair balances on different jobs

Captura de pantalla (1789)

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/peasants/gravedigger.dm [hunk 1]
```diff
@@ -39,6 +39,7 @@
 	if(H.mind)
 		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/polearms, 2, TRUE)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/peasants/gravedigger.dm:42]:
```suggestion
		H.mind.adjust_skillrank(/datum/skill/combat/polearms, 1, TRUE)
```
Non-combat class.

File: code/modules/jobs/job_types/roguetown/serfs/mason.dm [hunk 2]
```diff
@@ -50,7 +50,7 @@
 	beltr = /obj/item/storage/belt/rogue/pouch/coins/mid
 	beltl = /obj/item/rogueweapon/hammer/claw
 	backl = /obj/item/storage/backpack/rogue/backpack
-	backpack_contents = list(/obj/item/roguekey/mason = 1)
+	backpack_contents = list(/obj/item/roguekey/mason = 1, /obj/item/flint = 1, /obj/item/rogueweapon/huntingknife = 1, /obj/item/flashlight/flare/torch/lantern)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/serfs/mason.dm:53]:
```suggestion
	backpack_contents = list(/obj/item/roguekey/mason = 1, /obj/item/flint = 1, /obj/item/flashlight/flare/torch/lantern)
```
No free weapon for no particular reason at all. Backpacks also hold up to three items, not four.

File: code/modules/jobs/job_types/roguetown/serfs/mason.dm [hunk 3]
```diff
@@ -41,7 +41,7 @@
 	head = /obj/item/clothing/head/roguetown/hatfur
 	if(prob(50))
 		head = /obj/item/clothing/head/roguetown/hatblu
-//	armor = /obj/item/clothing/suit/roguetown/armor/workervest
+	armor = /obj/item/clothing/suit/roguetown/armor/leather/vest
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/serfs/mason.dm:44]:
```suggestion
	armor = /obj/item/clothing/suit/roguetown/armor/workervest
```
Not a combat class. No armor, only towner clothes.

File: code/modules/roguetown/roguejobs/gravedigger/tools.dm [hunk 4]
```diff
@@ -1,15 +1,19 @@
 /obj/item/rogueweapon/shovel
 	force = 5
-	force_wielded = 10
-	possible_item_intents = list(/datum/intent/mace/strike/shovel)
-	gripped_intents = list(/datum/intent/mace/strike/shovel, /datum/intent/shovelscoop)
+	force_wielded = 12
+	possible_item_intents = list(/datum/intent/mace/strike/shovel, /datum/intent/axe/chop/stone)
```
Comment (ThePainkiller) [inline code/modules/roguetown/roguejobs/gravedigger/tools.dm:4]:
```suggestion
	possible_item_intents = list(/datum/intent/mace/strike/shovel)
```
Chop intent on onehanded only for items that can't be dual handed. Read, hunting knife.

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/roguetown/serfs/mason.dm (hunk 2 above)
Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/serfs/mason.dm:53]: the knife goes for the reason of chop wood, isn't the same as an axe for round start but allows them to collect wood without craft a stone knife 
Captura de pantalla (1789)

the backpack can hold the starting items totally fine

Comment (ThePainkiller) [reply code/modules/jobs/job_types/roguetown/serfs/mason.dm:53]: They're masons, not carpenters.
They work stone and cogs, not wood.
Still a No on the hunting knife.

Comment (Dorodomki) [reply code/modules/jobs/job_types/roguetown/serfs/mason.dm:53]: i guess i got to map some axes for them to collect wood, dakkatown already has a few

**Comment by NPC1314** [comment]
This is mixed in a bit with #216, since it touches the same .dmi files

**Comment by Dorodomki** [comment]
> This is mixed in a bit with #216, since it touches the same .dmi files

unironically true, the inhands resprite should have been merged a while ago to keep the game updated and allow this to be merged quicker too :sadcat:

**Review by Stutternov — APPROVED** [review]

**Comment by NPC1314** [comment]
Check the icondiffbot, inhand shoveldirt in 64.dmi has a flicker issue, needs a hotfix

**Pull request merged**